src/smc-webapp/r_account.cjsx

```diff
@@ -475,9 +475,10 @@ AccountSettings = rclass
                 <Col xs=12>
                     <DeleteAccount
                         style={marginTop:'1ex'}
-                        initial_click={()=>@setState(show_delete_confirmation:true)}
-                        confirm_click={=>@props.redux.getActions('account').delete_account()}
-                        cancel_click={()=>@setState(show_delete_confirmation:false)}
+                        initial_click = {()=>@setState(show_delete_confirmation:true)}
+                        confirm_click = {=>@props.redux.getActions('account').delete_account()}
```

Comment:
There is a `get_fullname` method on the account store, which deals with some special cases (e.g., possibility of first or last name being undefined...)